source/includes/concept/_widget.md

```diff
@@ -0,0 +1,17 @@
+## Widget
+
+You have fetched your data from your [dataset](#dataset) using [queries](#query). And you know how to build custom visualizations for geospatial data using [layers](#layer). Yet, sometimes your data is not geo-located, and you want to build a plain graphical representation of it: this is when **widgets** come in handy.
+
+A **widget** is a visual specification of how to style and render the data of a dataset (think pie, bar or line charts).
+
+As with layers, each widget has a single dataset associated with it, and a dataset can be associated with many widgets. You can represent the same data in different ways by creating different widgets for the same dataset. The same widget can store independent configuration values for each RW API based application. It can also contain the required configuration for rendering the same visualization using different rendering tools.
+
+Like layers, the widget itself does not interact with the dataset data. You can either use the widget's `queryUrl` field to store the query to get the widget's data or store it inside the free form `widgetConfig` object. In any of these cases, it is your responsibility as an API user to query the data that will be used for rendering the widget.
```

Comment:
If the widget doesn't interact with dataset data, why do I have to associate widgets with datasets? If the answer is "for organizational purposes" we should say that.

Reply:
Added a line at the start of the paragraph stating that the association between datasets and widgets is only for organizational purposes.